packages/bpk-component-popover/src/BpkPopover.js

```diff
@@ -88,41 +88,22 @@ const BpkPopover = props => {
           className={getClassName('bpk-popover__arrow')}
           role="presentation"
         />
-        <div className={getClassName('bpk-popover__inner')}>
```

Comment:
This is just removing the `<div className={getClassName('bpk-popover__inner')}>` wrapper - nothing else has changed.